test/utility/bc-google-analytics.spec.js

```diff
@@ -154,4 +156,99 @@ describe('Baltimore County Google Analytics Utility', () => {
 			expect(actual).toEqual(true)
 		});
 	});
+
+	describe('getDefaultEvent', () => {
+		it('should return null if not a default action', () => {
+			const mockEvent = {
+				action: 'contact',
+				category: 'Branch Locations',
+				label: 'Arbutus'
+			};
+			const actual = googleAnalytics.getDefaultEvent(mockEvent);
+
+			expect(actual).toBeNull();
+		});
+		
+		it('should return an object with the key "method" and the value "Polaris"', () => {
+			const expected = {
+				method: 'Polaris'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'login',
+				label: 'Polaris'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the key "method" and the value "Twitter"', () => {
+			const expected = {
+				method: 'Twitter'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'share',
+				label: 'Twitter'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the key "search_term" and the value "book"', () => {
+			const expected = {
+				search_term: 'book'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'search',
+				label: 'book'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the key "search_term" and the value "book"', () => {
+			const expected = {
+				search_term: 'book'
+			};
+			const actual = googleAnalytics.getDefaultEvent({
+				action: 'view_search_results',
+				label: 'book'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+	});
+
+	describe('getStandardEvent', () => {
+		it('should return an object with the keys "event_category", "event_label" and the values "bbb", "ccc" respectively', () => {
+			const expected = {
+				event_category: 'bbb',
+				event_label: 'ccc'
+			};
+
+			const actual = googleAnalytics.getStandardEvent({
+				action: 'aaa',
+				category: 'bbb',
+				label: 'ccc'
+			});
+
+			expect(actual).toEqual(expected);
+		});
+
+		it('should return an object with the keys "event_category", "event_label", "value" and the values "bbb", "ccc", 1 respectively', () => {
+			const expected = {
+				event_category: 'bbb',
+				event_label: 'ccc',
+				value: 1
+			};
+
+			const actual = googleAnalytics.getStandardEvent({
+				action: 'aaa',
+				category: 'bbb',
+				label: 'ccc',
+				value: 1
+			});
+
+			expect(actual).toEqual(expected);
+		});
+	});
```

Comment:
Can we add a test for this scenario?

![image](https://user-images.githubusercontent.com/1933400/40781384-a8a55aaa-64a9-11e8-85b0-7d0bfdffbc36.png)

Reply:
Sure can

Reply:
@martypowell this is ready to be looked at.

Reply:
done